plugins/obs-webrtc/whip-output.cpp

```diff
@@ -21,6 +21,9 @@ const uint8_t audio_payload_type = 111;
 const char *video_mid = "1";
 const uint8_t video_payload_type = 96;
 
+// Convert Kbps to 'How many 1500 Byte packets in 5 seconds'
+const float videoNackStorageFactor = 125.0 / 1500.0 * 5.0;
```

Comment:
To resolve MSVC C4305 and C22220:

```suggestion
const float videoNackStorageFactor = 125.0f / 1500.0f * 5.0f;
```

Reply:
Changes were added (and was made constexpr)